Symfony/CS/Tests/Fixer/PSR2/BracesFixerTest.php

```diff
@@ -32,6 +32,20 @@ public function provideFixControlContinuationBracesCases()
         return array(
             array(
                 '<?php
+class Foo
+{
+    public function A()
+    {
+        ?>
+        Test<?php echo $foobar; ?>Test
+        <?php
+        $a = 1;
```

Comment:
When this line is removed, the fixer will add a white line.
The fix for this has a big impact on the performance of the fixer. If we've a transformer for the open tag this should not be an issue at all.